src/frontend/src/sass/_default.scss

```diff
@@ -74,7 +74,8 @@ h4 {
   padding-top: $padding-block;
   padding-bottom: calc($padding-block + $footer-height);
   display: flex;
-  justify-content: center;
+  justify-content: flex-start;
+  padding-left: 12rem;
 
```

Comment:
Instead of justifying content to the left and then applying a padding, we can justify content to the center and also apply a `margin: 0 auto` to give pages consistent spacing.

Reply:
Just following up. Is there a fix for the List page positioning incoming?